Description: Sanitize file/directory permissions and owner in libopts
 tarball. Override mtime with ${BUILD_DATE} which is exported in debian/rules.
 Avoid gzip timestamp.
Author: Andreas Metzler <ametzler@debian.org>
Origin: vendor
Forwarded: no
Last-Update: 2018-07-19

--- a/pkg/libopts/mklibsrc.sh
+++ b/pkg/libopts/mklibsrc.sh
@@ -123,7 +123,7 @@ echo ! tar cvf ${tag}.${sfx} ${tag}
 rbopts=""
 [ -z "$SOURCE_DATE_EPOCH" ] \
     || ! tar --help|grep -q sort= \
-    || rbopts="--sort=name --format=gnu --clamp-mtime --mtime @$SOURCE_DATE_EPOCH"
+    || rbopts="--sort=name --format=gnu --clamp-mtime --mtime @$SOURCE_DATE_EPOCH --owner=root --group=root --mode=u+w,g-w,a+r"
 
 tar cvf - $rbopts ${tag} | \
     $gz > ${top_builddir}/autoopts/${tag}.${sfx}
